File: Data Structures & Algorithms/remove-element/submission-0.py
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        vals = []
        numslength = len(nums)
        for i in nums:
            if i == val:
                vals.append(i)

        while val in nums:
            nums.remove(val)
        
        return numslength - len(vals)

File: Data Structures & Algorithms/sort-array-by-parity/submission-0.java
class Solution {
    public int[] sortArrayByParity(int[] nums) {
        int l = 0, r = nums.length - 1;
        while (l < r){
            if (nums[l]%2 != 0){
                int tmp = nums[l];
                nums[l] = nums[r];
                nums[r] = tmp;
                r--;
                l--;
            }
            l++;
        }
        return nums;
    }
}